tests/android/MASVS-CRYPTO/MASTG-TEST-0015.md

```diff
@@ -9,6 +9,8 @@ masvs_v1_levels:
 - L1
 - L2
 profiles: [L2]
+status: deprecated
+covered_by: ['MASTG-TEST-0xx1']
```

Comment:
add another placeholder test for the symmetric case (which is a different issue but it was previously part of this test)